src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -230,7 +229,6 @@ protected void verifyHashStoreProperties(
             );
 
```

Comment:
line 238 should be "store metadata namespace"

Reply:
Got it, thank you!